### ISSUE TITLE
Weird stack issue with recipes using empty honeycombs

### ISSUE THREAD

**Issue by Sunconure11** [GrowthcraftCE/Growthcraft-1.7#123]
I've noticed that if you use empty honeycombs in a recipe, and use more than one honeycomb at a time when only one is required, all of the honeycombs you used get sucked up, without creating multiple versions of the new product if the recipe meets the other requirements. I have a script in my modpack that lets you turn empty honeycombs from growthcraft into waxcombs from harvestcraft, and it kind of sucks up all of the items and only gives me one item in return, when it should give me several after inputting several empty honeycombs.

**Comment by IceDragon200** [comment]
@Sunconure11 weird, O: honeycombs are just your everyday item, they don't do anything special at all, can you give me steps to reproduce?

**Comment by Sunconure11** [comment]
Here is the script I am using with Harvestcraft. It requires Minetweaker.

http://pastebin.com/SCFGdDeH

Convert the empty honeycombs to waxcombs, and shift click on them.

**Comment by Sunconure11** [comment]
Something similar happens with drinks as well, if you drink a stacked beverage, it will only return one glass bottle.

**Comment by IceDragon200** [comment]
@Sunconure11 Ah, the drink bottles are an existing bug

``` java
return stack.stackSize <= 0 ? new ItemStack(Items.glass_bottle) : stack;
```

It will only add an empty bottle if the stack has been depleted

**Comment by IceDragon200** [comment]
@Sunconure11 Try this [build](https://dl.dropboxusercontent.com/u/72618186/minecraft/mods/growthcraft/2.3.1/growthcraft-1.7.10-2.3.1-complete--d2b8552.jar), I've fixed the bottle stack issue, not sure about the honeycombs though

**Comment by IceDragon200** [comment]
@Sunconure11 

What does `noReturn()` do here?

``` java
recipes.addShapeless(<harvestcraft:waxcombItem> * 1, [<Growthcraft|Bees:grc.honeyComb>.noReturn()]);
```

note: I've never used Minetweaker or Harvest Craft before

**Comment by Sunconure11** [comment]
It makes sure the item is consumed instead of returned, I had added it in because I thought it would stop this bug.

**Comment by IceDragon200** [comment]
@Sunconure11 Honeycombs are container items, they will return themselves when used in a recipe.

Now we make a decision, split the honeycomb into 2 separate items, the empty one and the filled one. 
That fixes your bug, but expect plenty of "breakage" afterwards.

**Comment by Sunconure11** [comment]
I suppose we can try that.

**Comment by IceDragon200** [comment]
@Sunconure11 Here, give this a try [build-c3c32f5](https://dl.dropboxusercontent.com/u/72618186/minecraft/mods/growthcraft/2.3.1/growthcraft-1.7.10-2.3.1-complete--c3c32f5.jar)

Also could you test the beebox for me, since I had to change quite a bit of code to get this in

The new honey comb items are:

```
honeyCombEmpty - replaces honeyComb:0
honeyCombFilled - replaces honeyComb:1
```

**Comment by Sunconure11** [comment]
I'll test it in a bit. What aspects of it need testing?

**Comment by IceDragon200** [comment]
@Sunconure11 Try placing a honey comb in the box, removing them, and see if it generates both honey combs and combs with honey with bees

tl;dr Make sure it does what it used to do

**Comment by Sunconure11** [comment]
All seems to be well. Now to let it generate honey, and then I'll confirm the results of this.

**Comment by Sunconure11** [comment]
It works.

**Comment by IceDragon200** [comment]
(ง •̀┌┐•́)ง Feel free to close unless you find any more issues
